### DISCUSSION TITLE
请教下， Desktop 能有个通俗易懂的文档看看吗

### DISCUSSION THREAD

**Discussion by zjxpcyc** [Serverless-Devs/Serverless-Devs#288] [Q&A] [answered]
先声明，我是新手。很多概念不懂。所以请教大佬们给点耐心。

首先，我说下对 Serverless-Devs 这个产品的理解，应该是帮助我们快速开发，部署，上线 Serverless 相关产品的一个东西。
基于这个认知，我决定用它来管理，我目前用到的几个 函数 FC。

1、工作空间，不清楚这个概念是什么。也没有找到地方新建。只是提示需要创建应用。

2、创建应用，我这边是想把原有项目改造为 FC 。一个方案是加入 FC 依赖，另外一个方案是 CustomRuntime . 我这边用的后者，这样我们一行代码都不需要修改。但是，好像目前来说没有找到支持这种方式的地方。

不知道是不是我有些先入为主了。我理想中的工具，不应该是那种，左边菜单是我的所有的服务或者应用列表。右边是对应服务或者应用的具体信息。比如 工作空间，组件中心，实用工具，在我的理解中，是我日常工作很低频用到的操作。

我想找的是，我在哪里可以管理我已经上线的 函数 FC

**Comment by anycodes** [comment, marked as answer]
感谢提问，我们关于Desktop的文档也正在紧张的编写中，目前我们的工作安排是这样的：

1. Serverless Devs的文档和工具重构升级 -> 文档已经完成了重构，工具正在升级中
2. FC组件的文档重构和升级 -> 文档正在重构
3. Desktop的文档建设和功能的升级 -> 待推进

我们也欢迎您如果有时间，有兴趣，可以和我们一起来做这一部分。咱们是一个开源建设的项目
